en/docs/administer/key-managers/configure-okta-connector.md

```diff
@@ -6,19 +6,19 @@ Follow the instructions below to configure Okta as a third-party Key Manager:
 
 ## Step 1 - Configure Okta
 
-1. Navigate to the [Okta sign up page](https://developer.okta.com/signup/).
+1. Navigate to the [Okta sign up page](https://developer.okta.com/signup/) and singup to **Workforce Identity Cloud**. We have created an account in **Developer Edition** for this.
 
      [![Okta signup]({{base_path}}/assets/img/administer/okta-signup.png)]({{base_path}}/assets/img/administer/okta-signup.png)
 
      Create an Okta account and get the Okta instance URL.
```

Comment:
Create an Okta account and get the Okta instance URL. You can create an account in **Developer Edition** for this.

Reply:
Shall we reword as above?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/8155/commits/387ce46364500909c31063133f186538f9120a48